api/grpcserver/mesh_service.go

```diff
@@ -567,3 +569,30 @@ func convertLayerStatus(in int) pb.Layer_LayerStatus {
 		return pb.Layer_LAYER_STATUS_UNSPECIFIED
 	}
 }
+
+func (s MeshService) EpochStream(req *pb.EpochStreamRequest, stream pb.MeshService_EpochStreamServer) error {
+	epoch := types.EpochID(req.Epoch)
+	atxids, err := s.mesh.EpochAtxs(epoch)
+	if err != nil {
+		return status.Error(codes.Internal, err.Error())
+	}
+	for i := 0; i < len(atxids); i += pageSize {
```

Comment:
@dshulyak 
i realized that pagination + stream is unusual. 
my original goal was to serve large set of data over RPC. but maybe this is an overkill.
if you'd like me to do a simple stream without repeated field in EpochStreamResponse, i'll change the api.

Reply:
yes, i think tech-wise pagination with streaming doesn't make much sense. the way TransactionResult works is that it load objects iteratively and writes it one by one to a stream, so max amount of memory that can be allocated by a stream is bounded by stream buffer.

in this change all atxs are loaded into memory and then paginated, not sure if there is anything good about that

Reply:
done to stream atx one by one. thanks for the help.